src/renderer/redux/actions/content.js

```diff
@@ -503,8 +506,8 @@ export function doPublish(params) {
 export function doAbandonClaim(txid, nout) {
   return function(dispatch, getState) {
     const state = getState();
-    const transactionItems = selectTransactionItems(state);
-    const { claim_id: claimId, claim_name: name } = transactionItems.find(
+    const myClaims = selectMyClaimsRaw(state);
```

Comment:
And we know this is always populated because `doFetchFileInfosAndPublishedClaims()` is called in `onDaemonReady()`?

Assuming yes, LGTM and can be merged.

Reply:
Yes, also `doFetchClaimListMine()` is also called at multiple important places such as after successful publish, visiting Published/Downloaded page etc. So it seems to keep updated information about claims.